pkg/controller/kogitoinfra/infinispan.go

```diff
@@ -44,14 +48,22 @@ const (
 	appPropInfinispanSaslMechanism
 	// appPropInfinispanAuthRealm application property for setting infinispan auth realm
 	appPropInfinispanAuthRealm
+	appPropInfinispanTrustStore
+	appPropInfinispanTrustStoreType
+	appPropInfinispanTrustStorePassword
 	// envVarInfinispanUser environment variable for setting infinispan username
 	envVarInfinispanUser
 	// envVarInfinispanPassword environment variable for setting infinispan password
 	envVarInfinispanPassword
 	infinispanEnvKeyCredSecret = "INFINISPAN_CREDENTIAL_SECRET"
 	enablePersistenceEnvKey    = "ENABLE_PERSISTENCE"
 	// saslPlain is the PLAIN type.
-	saslPlain string = "PLAIN"
+	saslPlain               string = "PLAIN"
```

Comment:
is this string type needed here?

Reply:
No need indeed. Must be a remaining clutter from other PRs. I'll remove it.